app/domain/authentication/authn_jwt/identity_provider_interface.rb

```diff
@@ -0,0 +1,10 @@
+module Authentication
+  module AuthnJwt
+    class IdentityProviderInterface
+      def initialize(authentication_parameters); end
+      def provide_jwt_identity; end
```

Comment:
add `authentication_parameters` as input to all other functions as well

Reply:
Why? Did you confused with the ConfigurationInterface?